docs/warnet.md

```diff
@@ -65,14 +65,24 @@ options:
 
 ### `warnet run`
 Run a scenario from a file.
-Pass `-- --help` to get individual scenario help
+    Pass `-- --help` to get individual scenario help
 
 options:
 | name            | type   | required   | default   |
 |-----------------|--------|------------|-----------|
 | scenario_file   | Path   | yes        |           |
 | additional_args | String |            |           |
 
+### `warnet run-binary`
+Run a file in warnet
+    Pass `-- --help` to get individual scenario help
```

Comment:
`-- --help` is specific to Commander/test framework, might not apply to all binaries